flask_security/core.py

```diff
@@ -214,6 +214,8 @@ def _request_loader(request):
     if request.get_json(silent=True):
         if isinstance(request.json, dict):
             token = request.json.get(args_key, token)
+        else:
+            abort(422)  # we can handle only object
```

Comment:
@jirikuncar 

But we have token candidate at line 213. Sorry for silly questions, I'm not familiar enough with this library yet.